demos/description-region/app.js

```diff
@@ -2,11 +2,11 @@ const toggleModeEl = document.getElementById("toggle-mode");
 const mapEl = document.querySelector("arcgis-map");
 const darkModeCss = document.getElementById("jsapi-mode-dark");
 const lightModeCss = document.getElementById("jsapi-mode-light");
-const liveRegion = document.getElementById("liveRegion");
 
+const liveRegion = document.getElementById("liveRegion");
 let mode = "light";
-
 toggleModeEl.addEventListener("click", handleModeChange);
+
```

Comment:
nitpick for line 29, do you think it would be useful to change the basemap when toggling between light/dark modes?

```js
document.body.className = isDarkMode ? "calcite-mode-dark" : "";
mapEl.basemap = isDarkMode ? "dark-gray" : "gray";
```

Reply:
Related nitpick:

What if the toggle switch also provided an aria-live region indicating the light/dark mode has been toggled?

Reply:
@geospatialem  let me look at the aria-live update for light/dark mode. I "think" that since the user clicks the button that tells them they are toggling the light/dark that aria-live might be overkill? But I can add it and see if it feels too chatty 
Updating the code to flip the basemap